docs/antora/modules/tailing/pages/tailing.adoc

```diff
@@ -360,3 +360,13 @@ Lastly, there is a script for running the reader named `queue_reader.sh` which a
 $ ./bin/queue_reader.sh <options>
 ----
 
+=== Refreshing The Directory Listing For A Queue Using `RefreshMain`
+
+`RefreshMain` can be used to refresh the directory listing for a queue, contained within the tablestore file for the queue. Assuming a project pom file that includes the Chronicle-Queue artifact, you can refresh the directory listing for a queue with the following command:
+
+[source, shell script]
+---
+mvn exec:java -Dexec.mainClass="net.openhft.chronicle.queue.main.RefreshMain" -Dexec.args="myqueue"
```

Comment:
there is also a shell script

Reply:
Is it possible that got left on a branch? I don't see it in the bin directory.

Reply:
sorry you are correct! Would you mind adding :) ?

Reply:
Not at all - I'll also add documentation for QueueUnlockMain as well, and create the top-level page for command line tools.